jupiter/openapi/jupiter-swagger.yaml

```diff
@@ -468,7 +473,6 @@ components:
       required: true
       schema:
         type: integer
-        format: int64
```

Comment:
@iamnivekx could you leave this as int64? It was introduced here: https://github.com/ilkamo/jupiter-go/issues/27